content/src/logic/database-queries/deployments-queries.ts

```diff
@@ -140,47 +173,41 @@ export function getHistoricalDeploymentsQuery(
       whereClause.push(SQL`dep1.entity_timestamp <= to_timestamp(${toEntityTimestamp} / 1000.0)`)
     }
   }
+}
 
-  if (filters?.deployedBy && filters.deployedBy.length > 0) {
-    const deployedBy = filters.deployedBy.map((deployedBy) => deployedBy.toLocaleLowerCase())
-    whereClause.push(SQL`LOWER(dep1.deployer_address) = ANY (${deployedBy})`)
-  }
-
-  if (filters?.entityTypes && filters.entityTypes.length > 0) {
-    const entityTypes = filters.entityTypes
-    whereClause.push(SQL`dep1.entity_type = ANY (${entityTypes})`)
-  }
-
-  if (filters?.entityIds && filters.entityIds.length > 0) {
-    const entityIds = filters.entityIds
-    whereClause.push(SQL`dep1.entity_id = ANY (${entityIds})`)
-  }
-
-  if (filters?.onlyCurrentlyPointed) {
-    whereClause.push(SQL`dep1.deleter_deployment IS NULL`)
-  }
-
-  if (filters?.pointers && filters.pointers.length > 0) {
-    const pointers = filters.pointers.map((p) => p.toLowerCase())
-    whereClause.push(SQL`dep1.entity_pointers && ${pointers}`)
-  }
-
-  let where = SQL``
-  if (whereClause.length > 0) {
-    where = SQL` WHERE `.append(whereClause[0])
-    for (const condition of whereClause.slice(1)) {
-      where = where.append(' AND ').append(condition)
-    }
+function getBaseQuery(includeOverwrittenInfo?: boolean) {
+  // TODO: make another query for when deleter deployment is needed
+  if (!!includeOverwrittenInfo) {
```

Comment:
aren't both queries equal and the only difference the left join?

I think it can be added as are the other clauses